WordPress/src/main/res/values/strings.xml

```diff
@@ -986,6 +986,30 @@
     <string name="editor_post_settings_set_featured_image">Set Featured Image</string>
 
 
+    <!-- Aztec -->
+    <string name="dialog_title">Insert link</string>
+    <string name="dialog_edit_hint">http(s)://</string>
+    <string name="dialog_button_ok">OK</string>
```

Comment:
This one and cancel below it can be dropped from both here and the editor lib - we're not using them anymore, as we switched to using `android.R.string.ok`/`cancel` (https://github.com/wordpress-mobile/WordPress-Android/pull/5651).